src/data/txn.lisp

```diff
@@ -725,9 +725,6 @@ pathname until a non-existant directory name has been found."
   (unix:unix-truncate (ext:unix-namestring pathname) position)
   #+sbcl
   (sb-posix:truncate (namestring pathname) position)
-  #+openmcl
-  (ccl:with-cstrs ((filename (namestring pathname)))
-    (#_truncate filename position))
```

Comment:
LW attempts to `read` #_truncate even though it's inside a feature macro. I don't think this is technically an LW bug, because LW does need to know where the expression blocked by #+openmcl ends. I don't know what #_truncate does so I don't know how to fix it correctly for openmcl.